src/System.Private.Xml/tests/Writers/XmlWriterApi/TCFullEndElement.cs

```diff
@@ -4270,7 +4270,18 @@ public void writeValue_27(XmlWriterUtils utils, int param, string sourceStr, str
                     if (param == 1)
                         w.WriteValue(value[sourceStr]);
                     else
-                        w.WriteAttributeString("a", value[sourceStr].ToString());
+                    {
+                        try
+                        {
+                            origCulture = CultureInfo.CurrentCulture;
+                            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;  // So that the number format doesn't depend on the current culture
+                            w.WriteAttributeString("a", value[sourceStr].ToString());
```

Comment:
Can InvariantCulture just be passed to ToString instead of changing the current one?